noir/noir-repo/compiler/noirc_evaluator/src/ssa/acir_gen/acir_ir/acir_variable.rs

```diff
@@ -2068,6 +2068,13 @@ impl<F: AcirField> AcirContext<F> {
         self.acir_ir.push_opcode(Opcode::Call { id, inputs, outputs, predicate });
         Ok(results)
     }
+
+    pub(crate) fn generate_assertion_message_payload(
+        &mut self,
+        message: String,
+    ) -> AssertionPayload<F> {
+        self.acir_ir.generate_assertion_message_payload(message)
+    }
```

Comment:
```suggestion
```

Not much point in having this if we only call it once.

Reply:
We call it from acir_gen, and we need that there since acir_ir is not a pub member ):